reglemente.tex

```diff
@@ -1569,17 +1639,17 @@ \section{Funktionärer}
 \end{reglemlista}
 
 %%% funktionar %%%
-\funktionar{SCalD}
+\funktionar{Skald}
 
-ScalD ansvarar för sångboken och ser till att den är uppdaterad och studentikos.
+Skald ansvarar för sångboken och ser till att den är uppdaterad och studentikos.
 
 \begin{reglemlista}
 
 	\item[Åligganden]
 	Det åligger SCalD
```

Comment:
```suggestion
	Det åligger Skald
```

Reply:
Har comittat, bör squahas in men vill ej rebasea nu